angular-viewer/adam-angular-demo/src/app/components/object-results/object-results.component.ts

```diff
@@ -26,6 +27,22 @@ export class ObjectResultsComponent implements OnChanges {
       features.push(feat);
     });
 
+    const affordances: Affordances[] = [];
+    if (entry.affordances) {
+      entry.affordances.forEach((element) => {
+        const element_split = element.split('_');
+        let feat = { name: element };
+        if (element_split.length === 4) {
+          feat = {
+            name: `can be ${element_split[0]} in "${element_split
+              .slice(1)
+              .join(' ')}"`,
+          }; // Formatting the affordances is already handled in the back end, so this is just here to be safe
```

Comment:
Nit, but I would rather put this before the statement -- either the line before the `feat = {` or on the line before the `if` -- I slightly prefer the latter but either works.

Reply:
I think putting it on the line before the `if` would look good, so I can move it there

Reply:
Done. I also realized that I had a strict equality check before:
`if (element_split.length === 4) {`
which would fail for affordances of the form `slot1_slot1_eats` or in an affordance descriptor with >2 slots, so I fixed that issue as well.

Reply:
Sounds good.